### PR TITLE
The cloner reminds you that you don't remember anything leading up to your death.

### PR DESCRIPTION
**Pull request by ghost**
## Changelog
:cl: BeloneX
tweak: The Cloner now reminds you that you don't remember anything that lead up to your death.
/:cl:

image

### CODE REVIEW

**Comment by Penwin0** [comment]
This should be MRP only since its only on MRP rulewise

**Comment by BeloneX** [comment]
> This should be MRP only since its only on MRP rulewise

You can just ignore it on Golden.

File: code/game/machinery/cloning.dm [hunk 1]
```diff
@@ -467,6 +467,7 @@
 	if(grab_ghost_when == CLONER_MATURE_CLONE)
 		mob_occupant.grab_ghost()
 		to_chat(occupant, "<span class='notice'><b>There is a bright flash!</b><br><i>You feel like a new being.</i></span>")
+		to_chat(occupant, "<span class='userdanger'><b>You do not remember anything about the events leading up to your death.</i></span>")
```
Comment (PowerfulBacon) [inline code/game/machinery/cloning.dm]: Opens with `<b>` closes with `</i>`

Comment (ghost) [reply code/game/machinery/cloning.dm]: I don't understand.

Comment (PowerfulBacon) [reply code/game/machinery/cloning.dm]: updated, didn't realize github formats it

File: code/game/machinery/cloning.dm [hunk 2]
```diff
@@ -467,6 +467,7 @@
 	if(grab_ghost_when == CLONER_MATURE_CLONE)
 		mob_occupant.grab_ghost()
 		to_chat(occupant, "<span class='notice'><b>There is a bright flash!</b><br><i>You feel like a new being.</i></span>")
+		to_chat(occupant, "<span class='userdanger'><b>You do not remember anything about the events leading up to your death.</b></span>")
```
Comment (PowerfulBacon) [inline code/game/machinery/cloning.dm]:
```suggestion
		to_chat(occupant, "<span class='userdanger'>You do not remember anything about the events leading up to your death.</span>")
```
userdanger is bold.
See stylesheet.dm
`.userdanger				{color: #ff0000;	font-weight: bold; font-size: 3;}`

**Comment by ike709** [comment]
Talking to Crossed about the viability of using a config for the RP level to easily restrict content such as this to its relevant server. DNM until then.

**Comment by francinum** [comment]
Here's your RP level fix.
@BeloneX look at how the policy.txt config works and use that. Add this to the sage config and so on.

**Comment by PowerfulBacon** [comment]
This has been on DNM since December going to assume it to just be dead. No point leaving it open

**Pull request closed without merging**